partcad/src/partcad/wrappers/wrapper_brep.py

```
@@ -0,0 +1,42 @@
+import os
+import sys
+
+from OCP.BRep import BRep_Builder
+from OCP.BRepTools import BRepTools
+from OCP.TopoDS import TopoDS_Shape
+
+sys.path.append(os.path.dirname(__file__))
+import wrapper_common
+
+
+def process(path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `process`

(ANN201)